install/index.markdown

```diff
@@ -134,37 +108,37 @@ Now follow the steps for your particular ROS version.
+    <div><div id="highlighter_230766" class="syntaxhighlighter nogutter  bash"><table border="0" cellpadding="0" cellspacing="0"><tbody><tr><td class="code"><div class="container"><div class="line number1 index0 alt2 highlighted"><code class="bash functions">source /opt/ros/indigo/setup.bash</code></div><div class="line number2 index1 alt1 highlighted"><code class="bash functions">mkdir moveit</code></div><div class="line number3 index2 alt2 highlighted"><code class="bash functions">cd moveit</code></div><div class="line number4 index3 alt1 highlighted"><code class="bash functions">mkdir src</code></div><div class="line number5 index4 alt2 highlighted"><code class="bash functions">cd src/</code></div><div class="line number6 index5 alt1 highlighted"><code class="bash plain">wstool init .</code></div><div class="line number7 index6 alt2 highlighted"><code class="bash plain">wstool merge https://raw.github.com/ros-planning/moveit_docs/indigo-devel/moveit.rosinstall</code></div><div class="line number8 index7 alt1 highlighted"><code class="bash plain">wstool update</code></div><div class="line number9 index8 alt2 highlighted"><code class="bash functions">cd ..</code></div></div></td></tr></tbody></table></div></div>
     <h4>STEP 3: Make sure MoveIt! dependencies are installed</h4>
-    <div><div id="highlighter_730265" class="syntaxhighlighter nogutter  bash"><table border="0" cellpadding="0" cellspacing="0"><tbody><tr><td class="code"><div class="container"><div class="line number1 index0 alt2 highlighted"><code class="bash plain">rosdep </code><code class="bash functions">install</code> <code class="bash plain">--from-paths src --ignore-src --rosdistro indigo -y</code></div></div></td></tr></tbody></table></div></div>
+    <div><div id="highlighter_730265" class="syntaxhighlighter nogutter  bash"><table border="0" cellpadding="0" cellspacing="0"><tbody><tr><td class="code"><div class="container"><div class="line number1 index0 alt2 highlighted"><code class="bash plain">rosdep install --from-paths src --ignore-src --rosdistro indigo -y</code></div></div></td></tr></tbody></table></div></div>
     <p></p></div>
     </div>
   </div>
 </div>
 
-
-#### **STEP 2 & 3: ROS Jade**
-
-<div class="accordion" id="accordion6">
+#### **STEP 2 & 3: ROS Hydro**
```

Comment:
I think we should just fully remove the hydro instructions (or move to a different "deprecated" page)